.github/CODEOWNERS

```diff
@@ -14,6 +14,8 @@ pnpm-workspace.yaml @pyth-network/web-team
 turbo.json @pyth-network/web-team
 .npmrc @pyth-network/web-team
 .nvmrc @pyth-network/web-team
+/.github/ @pyth-network/core-team @pyth-network/web-team
```

Comment:
I don't think the web team needs to be a codeowner for the whole `.github` directory, the web team is already listed as a codeowner for all the js related workflows below.  I can't recall how github deals with ordering for codeowners, but just make sure that the codeowners for the specific js-related actions below take precedence over the codeowners for the whole directory here.

Reply:
[Yes](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file), thank you
> Order is important; the last matching pattern takes the most precedence